src/controllers/therapists.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Request, Response } from 'express';
+import getAllTherapist from '../services';
+
+const getAllTherapists = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { q, page } = req.query;
+    const pageNumber = parseInt(page as string, 2) || 1;
+    const limit = 8;
+
+    const therapists = await getAllTherapist(q as string || ' ', pageNumber, limit);
+
+    if (!therapists.length) {
+      res.status(204).json({});
+    }
+
+    res.status(200).json(therapists);
```

Comment:
```suggestion
    res.json(therapists);
```
The default status code going to be `200` So there is no need to specify it, as well as response payload doesn't follow what we agreed on